olclient/schemata/import.schema.json

```
@@ -159,6 +163,17 @@
 	  "examples": ["duc d'Otrante"]
 	}
       }
+    },
+    "contributors": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["name"],
+      "properties": {
+        "role": { "type": "string",
```

Indent these like the rest of the file.
```suggestion
        "role": {
            "type": "string",
```